### ISSUE TITLE
111

### ISSUE THREAD

**Issue by zeimeiyou** [zeimeiyou/-#1]
loss = torch.tensor(0.0).to(device)
            for i in range(digit_num):
